website/blog/2024-03-22-pact-open-source-update-mar-2024.md

```diff
@@ -0,0 +1,282 @@
+      gunzip pact_verifier_cli-linux-$(uname -m).gz
+      mv pact_verifier_cli-linux-$(uname -m) pact_verifier_cli
+      ./pact_verifier_cli --help
+      ```
+
+    - [pact_mock_server_cli](https://github.com/pact-foundation/pact-reference/releases/tag/pact_mock_server_cli-v1.0.5)
+
+      ```sh
+      curl -LO https://github.com/pact-foundation/pact-reference/releases/download/pact_mock_server_cli-v1.0.5/pact_mock_server_cli-linux-$(uname -m).gz
+      gunzip pact_mock_server_cli-linux-$(uname -m).gz
+      mv pact_mock_server_cli-linux-$(uname -m) pact_mock_server_cli
+      ./pact_mock_server_cli --help
+      ```
+
+  - Whats left?
+    - The following projects require updating to build single linux binaries built with musl
+      - [pact-stub-server](https://github.com/pact-foundation/pact-stub-server)
+      - [pact-plugin-cli](https://github.com/pact-foundation/pact-plugins/tree/main/cli)
+      - [pact-protobuf-plugin](https://github.com/pactflow/pact-protobuf-plugin)
+      - [pact-csv-plugin](https://github.com/pact-foundation/pact-plugins/tree/main/plugins/csv)
+  - Pact FFI Alpine specific static and shared libraries
+    - These are now published, and require consumption in client libraries. Contributions are welcome, otherwise you'll hopefully see these rolling out throughout the year.
+
+Chat with us in: [#pact-rust](https://pact-foundation.slack.com/archives/CA2S7E6KC).
+
+<!-- ### Pact-Plugins
+
+Pact-Plugins: 
+
+- 
+
+Have you tried out Pact Plugins yet? What's stopping you?
+
+Chat with us in: [#pact-plugins](https://pact-foundation.slack.com/archives/CA2S7E6KC). -->
+
+<!-- ### .NET
+
+Pact-net: 
+
+- 
+
+Chat with us in: [#pact-net](https://pact-foundation.slack.com/archives/C9UTHV2AD). -->
+
+<!-- ### Golang
+
+Pact-go: 
+
+- 
+
+Chat with us in: [#pact-go](https://pact-foundation.slack.com/archives/C9UTHTFFB). -->
+
+### Node JS
+
+#### Pact-JS
+
+[Dany Marques](https://github.com/danymarques) went down the NPM rabbit hole, to investigate the root cause of a very interesting bug, affecting NPM registry proxies, and npm script-jacking!
+
+Some long nights with his detective hat on, saw Dany find a suitable workaround as an interim for the Pact-JS project, as well as upstream fixes to the npm project itself. We cannot thank you enough for your herculean efforts.
+
+- [PR](https://github.com/pact-foundation/pact-js-core/pull/494)
+- [Longest Slack thread ever xD - 213 replies!](https://www.linen.dev/s/pact-foundation/t/16633868/hell-all-quick-question-in-the-migration-guide-for-v12-it-s-#100081d7-2fee-4420-8eac-d55858dcc483)
+- Read more about manifest confusion in this [medium blog](https://medium.com/checkmarx-security/manifest-confusion-dont-believe-what-you-see-how-attackers-can-trick-developers-c3c6419f3ae4) (You may want to use incognito mode, if you don't have an account)
+
+Chat with us in: [#pact-js](https://pact-foundation.slack.com/archives/C9VBGLUM9).
+
```

Comment:
There were a few other changes too:

1. Corporate proxy support: https://github.com/pact-foundation/pact-js/pull/1199
2. Removed V4 feature toggle: https://github.com/pact-foundation/pact-js/pull/1176 (I've updated the docs to reflect this)

Maybe less critical
3. Fixed issue where matching rules not supported in messages: https://github.com/pact-foundation/pact-js/pull/1183

Reply:
I noted that 1) just got merged yesterday.

2) was mentioned last month https://docs.pact.io/blog/2024/02/19/pact-open-source-update-feb-2024#node-js

but happy to mention again, especially alongside the pact-php release

3) think it it worth pointing out the linked issues, and mentioning if you were blocked on that, it is now resolved and you can try again.


I also need to update pact-php to reflect yesterdays activities (mainlining of ffi branch and beta-7 release)